core-blocks/heading/editor.scss

```diff
@@ -15,6 +15,7 @@
 
 	h2 {
 		font-size: 1.95em;
+		line-height: 1.4;
```

Comment:
It seems weird to specify this here and for the `h2` below as well. Why is this one needed?

Reply:
That was ... not on purpose.